e2e_test.go

```diff
@@ -268,7 +268,7 @@ func TestE2E(t *testing.T) {
 			distribution = *d
 			break
 		}
-		time.Sleep(time.Second)
+		time.Sleep(cfg.TransactionPollInterval + time.Millisecond*100)
```

Comment:
Is `time.Millisecond*100` just an extra margin for waiting?

Reply:
yep exactly